src/blueprint/blueprint.rs

```diff
@@ -255,6 +256,11 @@ impl Blueprint {
 
         // We should safely assume the blueprint is correct and,
         // generation of schema cannot fail.
-        schema.finish().unwrap()
+        Ok(Valid::from(
+            schema
+                .finish()
```

Comment:
We can call `finish` in `from_config.rs` once before returning the blueprint. That will capture all the errors before hand thus not needing to remove this unsafe unwrap call.